modules/nginx.nix

```diff
@@ -0,0 +1,21 @@
+{ ... }:
+
+{
+  services.nginx = {
+    enable = true;
+    group = "shared";
+    appendConfig = ''
+      error_log /var/log/nginx/error.log debug;
+    '';
+  };
+
+  security.acme = {
+    acceptTerms = true;
+    defaults.email = "ymstnt@mailbox.org";
+  };
+
+  networking.firewall.allowedTCPPorts = [
+    80
+    443
+  ];
```

Comment:
This could be moved to a separate `networking.nix` module

Reply:
I'd keep it in nginx as that's the service that needs to open those ports. Most other services have an `openFirewall` option that do the same.

Reply:
Sure. Though I don't think we use the firewall currently, but I think we should consider in the future. This config can stay here, but general firewall related configs should be in `networking.nix`. Obviously, other service-related firewall config should go in the appropriate service module.

Reply:
If we need to set general firewall related configs, that can go into `networking` module. We are using a firewall as it is [enabled by default](https://search.nixos.org/options?channel=24.11&show=networking.firewall.enable&from=0&size=50&sort=relevance&type=packages&query=networking.firewall.enable)